python/minpy/__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
If we put minpy under python then there will be two packages to install. mxnet.minpy may be better.
Also since there won't be a minpy ndarray the separation may not make much sense. We can also do mxnet.autograd (for now it should be mxnet.contrib.autograd)